### ISSUE TITLE
should allow a user to specify any file path to run script

### ISSUE THREAD

**Issue by lloyd** [mozilla/chromeless#26]
currently we're creating a custom protocol for the data directory of the chromeless package inside main.js

This prevents us from being able to invoke run from the command line like so:

```
$ run ../mybrowser/
```

This also would make it difficult for us to handle breaking interface files into examples/ and tests/.

What would the downsides be of skipping the resource creation step and using file urls inside main.js?  this would let us get rid of the wierd symlinking we do in the run script?

**Comment by taboca** [comment]
From what I understand we put the protocol in first place because we did copy and paste of atul's code. The learning there, a benefit, was that we learned that the protocol module does accept "system" argument which governs the priviledges. If the file URL is working with relative path, i.e, the main HTML being able to load inner local pages, I am totally fine with it, but then we gotta make sure the file:// access does respect the security policy we are to set in place, which seems to be deny everything as we discussed.
